src/util/apiUtils.js

```diff
@@ -451,7 +451,10 @@ export const groupDuotoneSetter = duotoneUrls => group => {
 		);
 	const duotoneUrlRoot = duotoneKey && duotoneUrls[duotoneKey];
 	if (duotoneUrlRoot && photo.id) {
-		group.duotoneUrl = `${duotoneUrlRoot}/${photo.id}.jpeg`;
+		group.duotoneUrl = {
+			small: `${duotoneUrlRoot.small}/${photo.id}.jpeg`,
```

Comment:
photo scaler behavior - `.jpg` won't actually be generated/served for some reason